src/ostree/ot-builtin-static-delta.c

```diff
@@ -326,6 +330,22 @@ ot_static_delta_builtin_generate (int argc, char **argv, OstreeCommandInvocation
       if (opt_endianness || opt_swap_endianness)
         g_variant_builder_add (parambuilder, "{sv}", "endianness", g_variant_new_uint32 (endianness));
 
+      if (opt_key_ids) {
+        g_autoptr(GPtrArray) key_ids = g_ptr_array_new ();
+        for (char **iter = opt_key_ids; iter != NULL && *iter != NULL; ++iter)
+          g_ptr_array_add (key_ids, *iter);
+        g_autoptr(GVariant) key_ids_v = g_variant_new_strv ((const char *const *)key_ids->pdata,
+                                                            key_ids->len);
+        g_variant_builder_add (parambuilder, "{s@v}", "key-ids",
+                               g_variant_new_variant (g_steal_pointer (&key_ids_v)));
+      }
+      if (opt_sign_name)
+        g_variant_builder_add (parambuilder, "{sv}",
+                               "sign-name", g_variant_new_bytestring (opt_sign_name));
+      else
+        g_variant_builder_add (parambuilder, "{sv}",
+                               "sign-name", g_variant_new_bytestring ("ed25519"));
```

Comment:
Please use `if (!opt_sign_name) {opt_sign_name = "ed25519"; }` rather than calling the same function twice.